spec/models/inventory_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe Inventory, type: :model do
+  it 'has a valid factory' do
+    inventory = FactoryBot.build(:inventory)
+    expect(inventory).to be_valid
+  end
+
+  it 'is not valid without a name' do
+    inventory = FactoryBot.build(:inventory, name: nil)
+    expect(inventory).to_not be_valid
+  end
+
+  it 'is not valid with a name less than 3 characters' do
+    inventory = FactoryBot.build(:inventory, name: 'ab')
+    expect(inventory).to_not be_valid
+  end
+
+  it 'is not valid with a name longer than 50 characters' do
+    inventory = FactoryBot.build(:inventory, name: 'a' * 51)
+    expect(inventory).to_not be_valid
+  end
+
+  it 'is not valid without a description' do
+    inventory = FactoryBot.build(:inventory, description: nil)
+    expect(inventory).to_not be_valid
+  end
+
+  it 'belongs to a user' do
+    user_association = Inventory.reflect_on_association(:user)
+    expect(user_association.macro).to eq(:belongs_to)
+  end
+
+  it 'has many inventory_foods' do
+    inventory = Inventory.reflect_on_association(:inventory_foods)
+    expect(inventory.macro).to eq(:has_many)
+  end
+
+  it 'has many foods through inventory_foods' do
+    inventory = Inventory.reflect_on_association(:foods)
+    expect(inventory.macro).to eq(:has_many)
+    expect(inventory.options[:through]).to eq(:inventory_foods)
+  end
+end
```

Comment:
- The test are failing here too! Please, double check your test 

![image](https://github.com/Anand8317/recipe/assets/59397159/bf2598f4-585c-4a44-aed0-f264650d7794)

Reply:
### FAILING TEST 

- Please, fix all your tests as they fail across all files in the model.